lazy_tensor_core/test/test_lazy.py

```diff
@@ -0,0 +1,51 @@
+import torch
+import copy
+
+from torch.testing._internal.common_utils import run_tests, TestCase
+from torch.testing._internal.jit_utils import JitTestCase
+from torch.testing import FileCheck
+
+import lazy_tensor_core
+import lazy_tensor_core.debug.metrics as metrics
+
+lazy_tensor_core._LAZYC._ltc_init_ts_backend()
+torch.manual_seed(42)
+
+class TestLazyTensor(JitTestCase):
+        def testConvolutionBackward(self):
+                def clone_move(t):
+                        dev = 'lazy'
+                        copy_t = t.detach().clone().requires_grad_(True).to(device=dev)
+                        return copy_t
+
+                inp = torch.rand(1, 3, 128, 128, device='cuda', requires_grad = True)
+                inp_copy = clone_move(inp)
+                grad = torch.rand(1, 32, 121, 121, device='cuda') # no requires_grad
+                grad_copy = clone_move(grad)
+                weight = torch.rand(32, 3, 8, 8, device='cuda', requires_grad = True)
+                weight_copy = clone_move(weight)
+                bias = torch.rand(32, device='cuda', requires_grad = True)
+                bias = bias
+                bias_copy = clone_move(bias) 
+                bias_copy = bias_copy
+
+                # run eager
+                conv_out = torch.nn.functional.conv2d(inp, weight, bias)
+                (inp_grad, weight_grad, bias_grad) = torch.autograd.grad([conv_out], [inp, weight, bias], [grad])
+
+                # run lazy
+                conv_copy_out = torch.nn.functional.conv2d(inp_copy, weight_copy, bias_copy) 
+                (inp_copy_grad, weight_copy_grad, bias_copy_grad) = torch.autograd.grad([conv_copy_out], [inp_copy, weight_copy, bias_copy], [grad_copy])
+
+                jit_graph = lazy_tensor_core._LAZYC._get_ltc_tensors_backend([bias_copy_grad])
+                FileCheck().check("aten::sum(").run(jit_graph)
```

Comment:
you're checking to make sure the full graph is captured? and if the aten::sum is present, you assume that came from autograd and it means you captured everything?

Reply:
I was mostly checking that we captured the backward for the bias. There should be only one `sum` in the whole backward graph since there are no other broadcasts but the one on the bias.